### PR TITLE
Finish up Swift Challenges, Update: solution for Day 21 in Swift Language

### PR DESCRIPTION
**Pull request by NicolasCombe5555**
I saw in your issue #49 that someone has this assigned, but also the PR was closed. I made the day 22 challenge and tested it on hackerRank.

Reason on update:
You don't need to create a new generic type "Element" if the struct(on hackerRank question) already haves a generic type "T" that is also the naming convention for generic types.

### CODE REVIEW

**Comment by NicolasCombe5555** [comment]
@sagarchoudhary96 im sorry i didn't commented on the issue before working on this solutions, and that i didn't created a new branch for the PR. To compensate me not reading the contributing.md i improved it :) Happy Hacktoberfest!

File: Day22/Solution.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+func getHeight(root: Node?) -> Int {
+    	if root == nil || (root?.left == nil && root?.right == nil) {
```
Comment (sagarchoudhary96) [inline Day22/Solution.swift]: for this request you to update with below solution: 
```
func getHeight(root: Node?) -> Int {
    return root == nil ? -1 : 1 + max(getHeight(root: root!.left), getHeight(root: root!.right))
}
```

File: Day24/Solution.swift [hunk 2]
```diff
@@ -0,0 +1,11 @@
+func removeDuplicates(head: Node?) -> Node? {
+    var current = head
+    while current?.next != nil {
+        if current?.data == current?.next?.data {
```
Comment (sagarchoudhary96) [inline Day24/Solution.swift]: your solution is timing out when I'm submitting it on hackerrank, I request you to update it and test on hackerrank.

**Comment by sagarchoudhary96** [comment]
@NicolasCombe5555 please look at the above code changes required.  looking forward to updated PR soon :)

**Comment by NicolasCombe5555** [comment]
@sagarchoudhary96 Thats weird, i tried all of them yesterday and they passed the tests. I will check again in a few hours.

**Comment by NicolasCombe5555** [comment]
@sagarchoudhary96 I updated solution 22 as requested, thank you btw thats more swifty. As for the changed requested on day 24 i tested it again and i have no problems, can you try again? I took   a screenshot
Screen Shot 2019-10-27 at 4 10 10 PM

File: Day22/Solution.swift (hunk 1 above)
Comment (NicolasCombe5555) [reply Day22/Solution.swift]: Done!

**Pull request merged**